docs/lake/tutorials/listing.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Listing assets and adding pools
+sidebar_position: 6
+date: 2021-08-15 22:33:07
+---
+
+**[Pando Lake](https://lake.pando.im)** only allow for selected pools that have proven credentials. It not accept new pool from the public.
```

Comment:
Pando Lake only **allows** for... 

"It not accept" --> "It doesn't accept..."